xds/internal/client/load/store.go

```diff
@@ -24,8 +24,58 @@ import (
 
 const negativeOneUInt64 = ^uint64(0)
 
-// Store is a repository for LB policy implementations to report store load
-// data. It is safe for concurrent use.
+// A pair of cluster and service name. The same cluster can be used by multiple
+// services, and one service can use multiple clusters. So we need a pair with
+// both name to accurately indicate where the load belongs.
+type clusterAndServiceNames struct {
```

Comment:
If this type is only used as a map key, and does not necessarily translate into anything on the wire or actual load data, then how about providing a helper method which returns a `string` given a `cluster` and a `service`.

`func StoreKey(cluster, service string) string {...}`

And callers can use it as `pcls := ls.PerCluster(load.StoreKey(cluster, service))`.

Reply:
When recording loads, `cluster,service` is a pair.
But later when we update LRS stream to reports loads, it will only get loads for a cluster, not a serivce. So the type of key should be internal to the Store.

In fact, this struct is a temporary solution, because each LRS stream is responsible for a `cluster,service` pair.
Eventually, the store should be a `map[cluster]map[service]loads`

Reply:
Can we name this `storeKey` or something simpler like that?